schemas/littlehorse/service.proto

```diff
@@ -441,6 +458,34 @@ message RunWfRequest {
   optional WfRunId parent_wf_run_id = 6;
 }
 
+// Schedule WfRuns based on a specific cron UNIX expression
+message ScheduleWfRequest {
+  // Specific ID
+  optional string id = 1;
+  // The name of the WfSpec to run.
+  string wf_spec_name = 2;
+
+  // Optionally specify the major version of the WfSpec to run. This guarantees that
+  // the "signature" of the WfSpec (i.e. the required input variables, and searchable
+  // variables) will not change for this app.
+  optional int32 major_version = 3;
```

Comment:
How does it work if I add a new WfSpec version after the `ScheduledWfRun` is created?